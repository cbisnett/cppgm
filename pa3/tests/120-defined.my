0u
0u
0u
0u
0u
0u
0u
0u
0u
0u
0u
eof
