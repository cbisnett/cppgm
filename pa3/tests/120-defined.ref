0
1
0
1
error
error
0
1
0
1
error
eof
