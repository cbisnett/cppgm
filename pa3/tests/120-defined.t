defined foo
defined aha
defined bar
defined car
defined 123
defined

defined(foo)
defined (aha)
defined (  bar)
defined (car )
defined (123)
