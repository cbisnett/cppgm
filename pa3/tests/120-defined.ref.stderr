ERROR: non-identifier after defined in controlling expression
ERROR: non-identifier after defined in controlling expression
ERROR: non-identifier after defined in controlling expression
